toml.md

```diff
@@ -284,6 +284,8 @@ For convenience, some popular characters have a compact escape sequence.
 \UXXXXXXXX - unicode         (U+XXXXXXXX)
 ```
 
+Tab characters can be used either directly or in their escaped form `\t`.
```

Comment:
This is covered on line 277. I'm not sure what this clarification achieves.

Reply:
Hi @pradyunsg, thank you very much for the review.

I don't think line 277 covers the same content though.

Line 277 says that `\t` is a escape code for the tab character, but it does not explicitly say tab characters can be used directly also.

That in theory is covered by line 266. However it is hard to understand because of the double negative.

The idea of this addition is to explicitly state it can be used *both* ways, which is not explicitly stated elsewhere.

As a non native speaker, I had a hard time processing this info, so I was hoping to make it clearer.

Reply:
In other ways the information I am hoping to convey with that phrase is that people should not be afraid of using an "unescaped" tab char (which is not covered in line 277).